Strategy11/formidable-forms
# Introduce Form Templates Page

This PR introduces a new "Form Templates" page within the Formidable Admin Panel. The Form Templates page allows users to either start from scratch with a blank form or choose from a selection of pre-built templates.

## QA URL:
https://qa.formidableforms.com/sherv/wp-admin/admin.php?page=formidable-form-templates

## Testing Instructions:
1. Make sure you have Formidable PRO installed and activated on the **[`introduce-form-templates` branch](https://github.com/Strategy11/formidable-pro/tree/introduce-form-templates)**.
    - **Formidable PRO PR**: [https://github.com/Strategy11/formidable-pro/pull/4483](https://github.com/Strategy11/formidable-pro/pull/4483)
2. Confirm that the designs on the Form Templates page match those in the Figma file. **[Figma Design File](https://www.figma.com/file/x8Epd7CJLLY6JIyPoCMPvc/Create-Form?type=design&node-id=1864-17378&mode=dev)**
4. Review the video introductions (Part One and Part Two) and verify that every part of the feature works as shown.

## Video Introductions:
### Updated Custom Templates and Ready-Made Solutions
https://www.loom.com/share/4657fb8d47c2437d879e807ee851bacb?sid=93082411-a37c-4d31-a762-6cfc400418cb

### Part One
https://www.loom.com/share/d8ed343652dd4ed3b817a8d2d54ae284

### Part Two
https://www.loom.com/share/7fa9d217b98e448b8e1419531f86b710

## Checklist
- [x] Write PHPUnit Tests.
- [x] Test real licenses, including (Basic, Pro, Business, Elite, and expired)
